content/it/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,314 @@
+{{< /note >}}
+
+Di seguito i passi consigliati per configurare l'utilizzo di un registro privato da parte dei nodi del _cluster_. 
+In questo esempio, eseguire i seguenti comandi sul proprio desktop/laptop:
+
+   1. Esegui `docker login [server]` per ogni _set_ di credenziali che vuoi utilizzare. Questo comando aggiornerà `$HOME/.docker/config.json` sul tuo PC.
+   1. Controlla il file `$HOME/.docker/config.json` in un editor di testo per assicurarti che contenga le credenziali che tu voglia utilizzare.
+   1. Recupera la lista dei tuoi nodi; ad esempio:
+      - se vuoi utilizzare i nomi: `nodes=$( kubectl get nodes -o jsonpath='{range.items[*].metadata}{.name} {end}' )`
+      - se vuoi recuperare gli indirizzi IP: `nodes=$( kubectl get nodes -o jsonpath='{range .items[*].status.addresses[?(@.type=="ExternalIP")]}{.address} {end}' )`
+   1. Copia il tuo file locale `.docker/config.json` in uno dei path sopra riportati nella lista di ricerca.
+      - ad esempio, per testare il tutto: `for n in $nodes; do scp ~/.docker/config.json root@"$n":/var/lib/kubelet/config.json; done`
+
+{{< note >}}
+Per i cluster di produzione, utilizza uno strumento di gestione delle configurazioni per poter applicare le impostazioni su tutti i nodi laddove necessario.
+{{< /note >}}
+
+Puoi fare una verifica creando un Pod che faccia uso di un'immagine privata; ad esempio:
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: private-image-test-1
+spec:
+  containers:
+    - name: uses-private-image
+      image: $PRIVATE_IMAGE_NAME
+      imagePullPolicy: Always
+      command: [ "echo", "SUCCESS" ]
+EOF
+```
+```
+pod/private-image-test-1 created
+```
+Se tutto funziona correttamente, pochi istanti dopo, si può lanciare il comando:
+
+```shell
+kubectl logs private-image-test-1
+```
+e verificare che il comando restituisca in output:
+```
+SUCCESS
+```
+
+Qualora si sospetti che il comando sia fallito, si può eseguire:
+```shell
+kubectl describe pods/private-image-test-1 | grep 'Failed'
+```
+In caso di fallimento, l'output sarà simile al seguente:
+```
+  Fri, 26 Jun 2015 15:36:13 -0700    Fri, 26 Jun 2015 15:39:13 -0700    19    {kubelet node-i2hq}    spec.containers{uses-private-image}    failed        Failed to pull image "user/privaterepo:v1": Error: image user/privaterepo:v1 not found
+```
+
+Bisogna assicurarsi che tutti i nodi nel cluster abbiano lo stesso file `.docker/config.json`. 
+Altrimenti i pod funzioneranno correttamente su alcuni nodi ma falliranno su altri. 
+Ad esempio, se si utilizza l'autoscaling per i nodi, il template di ogni istanza 
+devono includere il file `.docker/config.json` oppure montare un disco che lo contenga.
+
+Tutti i pod avranno accesso in lettura alle immagini presenti in qualunque registro privato 
+una volta che le rispettive chiavi di accesso siano state aggiunte nel file `.docker/config.json`.
+
+### Immagini pre-scaricate
```

Comment:
```suggestion
### pre-pulled images
```

Reply:
Should be sentence case. Also, is “Immagini pre-pulled” a compromise?

Reply:
I think so too.. because if we start to use images in place of immagini it's really too English, in my opinion